settings.gradle

```diff
@@ -1 +1,3 @@
 include ':app', ':library'
+
+rootProject.name = 'chandelier'
```

Comment:
Could you try adding that to the module directly?

Reply:
Everything looks good. 

https://jitpack.io/#prolificinteractive/chadelier